colors/nord.vim

```diff
@@ -684,3 +684,13 @@ hi! link VimwikiList markdownListMarker
 " YAML
 " > stephpy/vim-yaml
 call s:hi("yamlKey", s:nord7_gui, "", s:nord7_term, "", "", "")
+
+" Mail
+call <sid>hi("mailQuoted1",  s:nord3_gui, "", s:nord3_term, "", "", "")
```

Comment:
This could be simplified like this:

```vimscript
for s:mailquote_i in range(1, 6)
  call hi("mailQuoted" . s:mailquote_i,  s:nord3_gui, "", s:nord3_term, "", "", "")
endfor
```